File: ex3.rb
RUBY_VERSION                    # => "2.7.0"

# Enumerable#inject を使わずに Array の総乗を求めよう。

a = [1, 2, 3, 5]

result = 1
a.each do |i|
  result *= i
end

result                          # => 30

File: ex3.cs
using System;
using System.Collections.Generic;
using System.Linq;

// Enumerable#inject を使わずに Array の総乗を求めよう。

class RubyKansai
{
    static void Main()
    {
        var a = new[]{1, 2, 3, 5};

        var result = a.Aggregate(1, (s, i) => s * i);

        Console.WriteLine($"{result}");
    }
}
